### PR TITLE
Server setup: correct statement after Hub

### PR DESCRIPTION
**Pull request by moio**
# Description

This corrects a statement which is slightly different after the introduction of the Hub architecture.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

### CODE REVIEW

File: modules/installation/pages/server-setup.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ As part of providing product support, SUSE reserves the right to ask to remove a
 [IMPORTANT]
 ====
 Do not register a {productname} Server to itself.
-The {productname} Server must be managed individually.
+The {productname} Server must be managed individually or via another separate {productname} Server.
```
Comment (Loquacity) [inline modules/installation/pages/server-setup.adoc:24]:
```suggestion
The {productname} Server must be managed individually or by using another separate {productname} Server.
For more information about using multiple servers, see xref:large-deployments:multi-server.adoc[].
```

Comment (Loquacity) [reply modules/installation/pages/server-setup.adoc:24]: Also: this statement exists in multiple locations, we'll need to update them globally.

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by keichwa — APPROVED** [review]

File: modules/installation/pages/server-setup.adoc (hunk 1 above)
Comment (moio) [reply modules/installation/pages/server-setup.adoc]: > Also: this statement exists in multiple locations

Fixed

**Comment by moio** [comment]
@keichwa, @Loquacity: I changed commits now to patch all mentions I could find. Note that I have removed some as they did not make sense to me, like warning not to register a Server onto itself in the Ubuntu section (Servers can only be on SLE or openSUSE).

Hope it's fine now!

**Review by keichwa — APPROVED** [review]
Sounds sensible.  Thanks a lot, Silvio @moio !

**Review by Loquacity — APPROVED** [review]

**Comment by moio** [comment]
Please note this needs cherry-picking to 4.1 and 4.2!

**Comment by Loquacity** [comment]
:heavy_check_mark: Backported to `manager-4.1`

**Pull request merged**